### PR TITLE
Cleanup some Tech Debt

### PR DESCRIPTION
**Pull request by marshallswain**
- Added tests for page-level components.
- Uses tabs for indentation.
- Some unused components have been removed.
  - `components/login/` was deleted.
  - `components/register/` was deleted.
  - `models/signup` was deleted.

### CODE REVIEW

**Comment by justinbmeyer** [comment]
In general we shouldn't use component files for anything that has a viewModel.

**Comment by marshallswain** [comment]
> In general we shouldn't use component files for anything that has a viewModel.

Is that due to testing purposes?  

Two of those components have viewModels with no custom logic in them.  One depends on attributes being passed in and the other accepts no attributes, but contains a single `openPopup` function.  Do you consider it worth it to create a full modlet for those?  Maybe I can instead put a test next to the `.component` file and import the viewModel.  I didn't know you could do that until I was trying to fix tests for https://github.com/donejs/done-component/pull/21 last night.

**Comment by justinbmeyer** [comment]
> Is that due to testing purposes? Or is there something else in consideration?

Testing, demo pages and code highlighting are considerations.  Also, what happens if the code "expands"?  Modlets are the way to go unless the component is pretty much entirely HTML + CSS.

> consider it worth it to create a full modlet for those

Yeah, I consider it worth it.

File: public/components/page-contributors/page-contributors.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <script type="text/stache" can-autorender>
-  <can-import from="bitcentive/components/register/" />
-  <bit-register></bit-register>
+  <can-import from="bitcentive/components/page-contributors/" />
+  <page-contributors></page-contributors>
```
Comment (justinbmeyer) [inline public/components/page-contributors/page-contributors.html]: `<page-contributors/>`

**Pull request merged**